react-toggle-switch-component/my-app/src/ToggleSwitch.tsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from 'react';
+import './ToggleSwitch.css';
+
+export default function ToggleButton() {
+  const [toggle, setToggle] = useState(false);
+  const [toggleText, setToggleText] = useState('OFF');
+  const [active, setActive] = useState('');
```

Comment:
I wouldn't store these in state. It's causing this component to re-render more times than needed. I would just use a ternary to assign the variable a value based off of the value of `toggle`.

`const toggleText = toggle ? 'OFF' : 'ON'`

Reply:
That makes sense, I didn't think about the fact that each instance of `useState` would cause a rerender.